x-pack/plugins/security_solution/server/lib/detection_engine/signals/rule_status_service.mock.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SavedObjectsClientContract } from '../../../../../../../src/core/server';
+import { RuleStatusSavedObjectsClient } from './rule_status_saved_objects_client';
+import { RuleStatusService } from './rule_status_service';
+
+export type RuleStatusServiceMock = jest.Mocked<RuleStatusService>;
+
+export const ruleStatusServiceFactoryMock = async ({
```

Comment:
I don't think this file is used anymore? If not, you want to delete it? You can keep it if you think these mocks will be useful for someone else though. Just pointing it out.